leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -345,7 +345,10 @@ private void proposeLeadership(LeadershipToken token) {
             proposer.propose(seq, null);
         } catch (PaxosRoundFailureException e) {
             // We have failed trying to become the leader.
-            leaderLog.warn("Leadership was not gained", e);
+            leaderLog.warn("Leadership was not gained because: " + e.getMessage() + ". "
```

Comment:
I'd be fearful of doing that at the expense of people having the logs they need. I think a thing we talked about was having more information available at the debug level.